ios/browser/api/sync/brave_sync_worker.cc

```diff
@@ -318,7 +318,7 @@ bool BraveSyncWorker::IsSyncEnabled() {
     return false;
   }
 
-  return setup_service->IsSyncEnabled();
+  return setup_service->CanSyncFeatureStart();
```

Comment:
`BraveSyncWorker::IsSyncEnabled` -> `CanSyncFeatureStart` for consistency

Reply:
Addressed in 70e2a74dc8d246dab6e40d946e90f17be9e42d3c